lib/services/files-api-service.js

```diff
@@ -0,0 +1,129 @@
+// Copyright 2016, EMC, Inc.
+
+'use strict';
+
+var di = require('di');
+var fs = require('fs');
+var path = require('path');
+var walking = require('walk');
+
+module.exports = filesApiServiceFactory;
+
+di.annotate(filesApiServiceFactory, new di.Provide('Http.Services.Api.StaticFiles'));
+di.annotate(filesApiServiceFactory,
+    new di.Inject(
+      'Constants',
+      'Promise',
+      'Services.Configuration',
+      'Services.Waterline'
+    )
+);
+
+
+function filesApiServiceFactory(
+    constants,
+    Promise,
+    configuration,
+    waterline
+) {
+    var walker;
+    var walk = Promise.promisifyAll(walking);
+    var skuPath = constants.HttpStatic.onhttpSkupack;
+    var staticPath = constants.HttpStatic.onhttpStaticHttp;
```

Comment:
Put a link to the PR in your description here when you do, and make sure to say "requires <link>" otherwise we don't have a way of quickly knowing that we _shouldn't_ merge this PR until the other one goes in.